src/hooks.spec.tsx

```diff
@@ -272,7 +278,6 @@ describe('hooks', () => {
 
       //   // TODO - Wrap this with async act() once we upgrade to React 16.9
       //   // See https://github.com/facebook/react/issues/15379
```

Comment:
I feel we should clean up comments. 

It looks like `act` is deprecated, so we'll need a new work ticket soon to clean up our uses of it.